response_operations_ui/templates/account/my-account.html

```diff
@@ -12,13 +12,14 @@
 <section>
     <h1 id="respondent-name">Account details</h1>
     <dl class="ons-metadata ons-metadata__list ons-grid ons-grid--gutterless ons-u-cf ons-u-mb-l" title="">
-        <dt class="ons-metadata__term ons-grid__col ons-col-4@m">Email address:</dt>
-        <dd class="ons-metadata__value ons-grid__col ons-col-8@m">{{ user.email }}</dd>
-        <dt class="ons-metadata__term ons-grid__col ons-col-4@m">Name:</dt>
-        <dd class="ons-metadata__value ons-grid__col ons-col-8@m">{{ user.name }}</dd>
-        <dt class="ons-metadata__term ons-grid__col ons-col-4@m">Password:</dt>
-        <dd class="ons-metadata__value ons-grid__col ons-col-8@m">●●●●●●●●</dd>
-        <dd class="ons-metadata__value ons-grid__col ons-col-8@m ons-u-ml-no ons-u-mb-xs ons-u-fs-s">(Last changed on {{ user.password_last_changed }})</dd>
+        <dt class="ons-metadata__term ons-grid__col ons-col-4@m"><strong>Email address:</strong></dt>
```

Comment:
Off the back of the fronstage PR have made these changes as well. This keeps the same visual format as before it seems ons-metadata__term was adding bold text styling and is no longer doing that.